bodhi/server/models.py

```diff
@@ -317,6 +317,16 @@ class UpdateStatus(DeclEnum):
     processing = 'processing', 'processing'
 
 
+class CiStatus(DeclEnum):
+    """ This class lists the different status the ci_status flag can have.
+    """
+    ignored = 'ignored', 'ignored'
+    queued = 'queued', 'queued'
+    running = 'running', 'running'
+    passed = 'passed', 'passed'
+    failed = 'failed', 'failed'
```

Comment:
For each of these, the second one is meant to be used for a human-readable label. I recommend capitalizing each of them. I know that's not consistent with the others here, but I've been meaning to fix them, and the new one I just added for type uses the human-readable form.